docs/usage.md

```
@@ -0,0 +1,486 @@
+directory, executes the `actionlint` command on any detected GitHub Actions files within the directory, and reports returned
+information in the code editor display tab while editing workflow files.
+
+### Nova
+
+[Nova.app][nova] is a MacOS only editor and IDE. The [Actionlint for Nova][nova-extension] allows you to get inline feedback
+while editing actions.
+
+### trunk
+
+[trunk][trunk-io] is an extendable superlinter with a builtin language server and preexisting issue detection. Actionlint is
+integrated [here](https://github.com/trunk-io/plugins).
```

markdownlint-cli2 (0.21.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)